static/src/javascripts/projects/common/modules/commercial/commercial-features.js

```diff
@@ -97,6 +97,7 @@ class CommercialFeatures {
             !isHosted &&
             !isInteractive &&
             !config.get('page.isFront') &&
+            !config.get('isDotcomRendering', false) &&
```

Comment:
This code reads as though DCR doesn't support or allow this slot. Might be worth adding a note to explain that DCR already renders it server side so we don't need this client side step?